doc/visualizations.rst

```diff
@@ -8,37 +8,85 @@ Scikit-learn defines a simple API for creating visualizations for machine
 learning. The key feature of this API is to allow for quick plotting and
 visual adjustments without recalculation. We provide `Display` classes that
 expose two methods for creating plots: `from_estimator` and
-`from_predictions`. The `from_estimator` method will take a fitted estimator
-and some data (`X` and `y`) and create a `Display` object. Sometimes, we would
-like to only compute the predictions once and one should use `from_predictions`
-instead. In the following example, we plot a ROC curve for a fitted support
-vector machine:
+`from_predictions`.
+
+The `from_estimator` method generates a `Display` object from a fitted estimator,
+input data (`X`, `y`), and a plot.
+The `from_predictions` method creates a `Display` object from true and predicted
+values (`y_test`, `y_pred`), and a plot. Using `from_predictions` avoids having to
+recompute predictions, but the user needs to take care that the prediction values
+passed correspond to the `pos_label`.
+
+For :term:`predict_proba`, the column corresponding to the probability estimate of
+the `pos_label` class is selected while for :term:`decision_function`, the score is
+reverted (i.e. multiply by -1) when `pos_label` is not the label 1.
```

Comment:
Again, I don't know if 0,1 / -1,1 are always the classification labels? Could it not be e.g., 0 and 2 ? In which case it does not make sense when you say > `pos_label` is not the label 1. See the decision function glossary entry: https://scikit-learn.org/stable/glossary.html#term-decision_function

```suggestion
For :term:`predict_proba`, select the column corresponding to the
the `pos_label` class while for :term:`decision_function`, revert the
score (i.e. multiply by -1) if `pos_label` is not the last class in the
`classes_` attribute of your estimator.
```

The wording for decision function is a bit tricky honestly. Maybe it could be improved but is technically correct.